app/(app)/_layout.tsx

```diff
@@ -2,11 +2,13 @@ import { useMedplum } from "@medplum/react-hooks";
 import { Redirect, Slot } from "expo-router";
 import { SafeAreaView } from "react-native-safe-area-context";
 
+import { PractitionerBanner } from "@/components/PractitionerBanner";
 import { Spinner } from "@/components/ui/spinner";
 import { ChatProvider } from "@/contexts/ChatContext";
 
 export default function AppLayout() {
   const medplum = useMedplum();
+  const isPractitioner = medplum.getProfile()?.resourceType === "Practitioner";
```

Comment:
No need to change, but it could also be:

- alternative 1

```suggestion
  const medplum = useMedplum();
  const profile = useMedplumProfile();
  const isPractitioner = profile?.resourceType === "Practitioner";
```

- alternative 2

```suggestion
  const { medplum, profile } = useMedplumContext();
  const isPractitioner = profile.resourceType === "Practitioner";
```